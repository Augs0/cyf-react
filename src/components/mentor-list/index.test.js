import React from 'react';
import { shallow } from 'enzyme';

import MentorList from './';

jest.mock('uuid', () => {
  return {
    v4: jest.fn(() => 1),
  };
});

describe('MentorList Component', () => {
  it('renders correctly', () => {
    expect(shallow(<MentorList />)).toMatchSnapshot();
  });
});
