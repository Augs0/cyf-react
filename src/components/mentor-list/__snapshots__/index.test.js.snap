// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MentorList Component renders correctly 1`] = `
<Styled(section)
  id="mentors"
>
  <Styled(MentorCard)
    firstName="GermÃ¡n"
    id="german_bencci"
    key="german_bencci"
    lastName="Bencci"
    photo="german.jpg"
    title="Organiser"
  />
  <Styled(MentorCard)
    firstName="Mozafar"
    id="mozafar_haider"
    key="mozafar_haider"
    lastName="Haider"
    photo="mozafar.jpg"
    title="Scotland Organiser & React"
  />
  <Styled(MentorCard)
    firstName="Kash"
    id="kash_karimi"
    key="kash_karimi"
    lastName="Karimi"
    photo="kash.jpg"
    title="Javascript & React"
  />
  <Styled(MentorCard)
    firstName="Amit"
    id="amit_shah"
    key="amit_shah"
    lastName="Shah"
    photo="amit.jpg"
    title="Javascript and Final Project"
  />
  <Styled(MentorCard)
    firstName="Simon"
    id="simon_legg"
    key="simon_legg"
    lastName="Legg"
    photo="simon.jpg"
    title="Javascript and Final Project"
  />
  <Styled(MentorCard)
    firstName="Bex"
    id="bex_bolton"
    key="bex_bolton"
    lastName="Bolton"
    photo="bex.jpg"
    title="Javascript & NodeJS"
  />
  <Styled(MentorCard)
    firstName="Claudia"
    id="claudia_matosa"
    key="claudia_matosa"
    lastName="Matosa"
    photo="claudia.jpg"
    title="HTML, CSS & React"
  />
  <Styled(MentorCard)
    firstName="James"
    id="james_wright"
    key="james_wright"
    lastName="Wright"
    photo="james.png"
    title="NodeJS and Databases"
  />
  <Styled(MentorCard)
    firstName="Nate"
    id="nate_wright"
    key="nate_wright"
    lastName="Wright"
    photo="nate.jpg"
    title="HTML & CSS"
  />
  <Styled(MentorCard)
    firstName="Will"
    id="will_wright"
    key="will_wright"
    lastName="Wright"
    photo="will.jpg"
    title="HTML, CSS & Javascript"
  />
  <Styled(MentorCard)
    firstName="Rob"
    id="rob_gelb"
    key="rob_gelb"
    lastName="Gelb"
    photo="rob.jpg"
    title="Employment, Company Links"
  />
  <Styled(div)>
    <Styled(Link)
      className="big-link-3 btn"
      replace={false}
      to="/volunteers"
    >
      Become a volunteer
    </Styled(Link)>
  </Styled(div)>
</Styled(section)>
`;
